src/openfl/text/TextField.hx

```diff
@@ -3059,6 +3063,55 @@ class TextField extends InteractiveObject
 			scrollV -= event.delta;
 		}
 	}
+	
+	@:noCompletion private function this_onDoubleClick(event:MouseEvent):Void
+	{
+		if (__textEngine.selectable) {
```

Comment:
I would do `selectable` (i.e. `this.get_selectable()`), just in case the getter ever needs to be modified